### PR TITLE
Merge pull request #1300 from pythonitalia/feature/newsletter

### PR DESCRIPTION
**Pull request by estyxx**
### TODO 
- - [ ] Understand how to pass email to unsubscribe to the FE [maybe pinpoint can help us]
- - [ ] Check the unsubscriber page message, now there is a simple and cold  "You have been successfully unsubscribed from this newsletter. ", do we want to put something like "we are sorry..." or "Did you click on unsubscribe by mistake? Click here to subscribe again!"... 
- - [ ] Retrieve these list of recipients: 
-  - [ ] Speakers without tickets
   -  [ ] Users that did not fill their ticket
   -  [ ] all users with ticket 
   -  [ ] Speakers that did not fill their ticket
- - [ ] Send list of subscriber to amazon pinpoint

### CODE REVIEW

**Comment by marcoacierno** [comment]
Do we want to integrate with Mailchimp?

**Comment by patrick91** [comment]
@marcoacierno I think we should migrate away from it, as it is quite expensive. We now have a big limit with SES, so we could use that to send emails, but we need to build something on django I guess

**Comment by estyxx** [comment]
I think that this PR could be already merged. I add a component to keep the list of subscribers (registered user and not) and add a little form in the homepage in order to allow people to subscribe. 
We will have to integrate AWS and build a little tool (in the admin panel maybe) to allow us to send emails to a selected list of users (newsletter subscribers, but also specific user) see my comment [#1090](https://github.com/pythonitalia/pycon/issues/1090).

**Comment by estyxx** [comment]
The PR it's almost complete... but continue to throw this error (in checks and locally): 
Screenshot 2019-12-09 at 22 12 25

I don't understand, seems there is some problem in that query when gatsby tries to execute in the start script
```
  const result = await graphql(
    `
      query {
        backend {
          blogPosts {
            slugEn: slug(language: "en")
            slugIt: slug(language: "it")
          }
          pages {
            slugEn: slug(language: "en")
            slugIt: slug(language: "it")
          }
        }
      }
    `,
  );
```

I have at least one blogPost and a Page, so there is something...

**Comment by estyxx** [comment]
I added the support to send email from the admin panel. Maybe the template for the newsletter needs to be updated (now is the same of the change-password email...). 
And there is _that error_ I mentioned above, I don't understand what is happening...

File: backend/api/newsletters/forms.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+class UnsubscribeToNewsletterForm(FormWithContext):
+    email = forms.EmailField()
+
+    def save(self):
+        email = self.cleaned_data.get("email")
+        try:
+            deleted, _ = Subscription.objects.get(email=email).delete()
+            return deleted == 1
```
Comment (patrick91) [inline backend/api/newsletters/forms.py]: we can just have this, `Subscription.objects.get(email=email).delete()` no need to check :)

**Pull request merged**